scripts/bootstrap-py3-venv

```diff
@@ -24,7 +29,10 @@ command -v direnv >/dev/null || {
 
 gitroot="$(git rev-parse --show-toplevel)"
 cd "$gitroot"
-export venv_name="${PWD}/.venv"
+# This executes very quickly if the Python versions are already installed
+make setup-pyenv
+export PYENV_VERSION=$(SENTRY_NO_VENV_CHECK=1 ./scripts/do.sh get-pyenv-version)
```

Comment:
I need to set the variable so `pyenv` will know what Python version to change to.
I've tested this on both my machines.